src/data/major-projects.js

```diff
@@ -27,6 +27,23 @@ const majorProjects = [
       { label: "Website", url: "https://cilium.io" },
     ],
   },
+  {
+    logoUrl: "https://github.com/pyroscope-io/pyroscope",
```

Comment:
The stated requirements for being listed at ebpf.io include:
"The project must be using eBPF as its underlying core technology, in other words, a project would lose its purpose if the eBPF parts are removed."

However [when I look at https://github.com/pyroscope-io/pyroscope there is a ](https://github.com/pyroscope-io/pyroscope#deployment-diagram) shows a number of deployment agents of which ebpf is only one of them.  To me, this means pyroscope itself fails the requirement and should not be listed on this page.  Instead, the "(eBPF) Linux Kernel Pyroscope Agent" can be listed, but not the platform per se.   But then I suspect the ebpf pyroscope agent hasn't reached 50 contributors so it would not be major, only emerging.

Reply:
Correct, my reading is that it does not meet the requirements to have been listed there in the first place.

Reply:
@borkmann since you merged the original PR, would you agree?